test/hack/update-bicep-json.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+project_root="$(dirname "${BASH_SOURCE}")/../.."
+
+ACTUAL_OUTPUT_DIR="${project_root}/test/e2e/test-artifacts/generated-test-artifacts"
+OUTPUT_DIR="${1:-${ACTUAL_OUTPUT_DIR}}"
+
+# clean the output directory
+rm -rf "${OUTPUT_DIR}/*"
+
+az bicep build --file="${project_root}"/demo/bicep/customer-infra.bicep --outfile="${OUTPUT_DIR}"/standard-cluster-create/customer-infra.json
```

Comment:
Not clear to me this needs to be checked in - any reason it can't be done on the fly by the test framework?